src/arvbuffer.c

```diff
@@ -399,9 +399,20 @@ arv_buffer_get_frame_id (ArvBuffer *buffer)
 	return buffer->priv->frame_id;
 }
 
-void arv_buffer_set_frame_id(ArvBuffer *buffer, guint64 frameId)
+/**
+ * arv_buffer_get_frame_id:
+ * @buffer: a #ArvBuffer
+ *
+ * Sets the buffer frame id. For GigEVision devices, valid values are in the
+ * 1..65535 range. Useful for creating a replay fake camera.
+ *
+ * Since: 0.8.0
+ */
+
+void
+arv_buffer_set_frame_id(ArvBuffer *buffer, guint64 frame_id)
 {
```

Comment:
Parameters should be checked using g_return_if_fail helpers (ARV_IS_BUFFER and frame_id range)

Reply:
I don't believe there is any restriction on the frame_id range?